media/css/mozorg/technology.scss

```diff
@@ -0,0 +1,881 @@
+                        text-align: left;
+                    }
+                }
+            }
+        }
+    }
+
+    @media #{$mq-desktop} {
+
+        .content {
+            min-height: 450px;
+        }
+
+        .inner-container {
+            width: 440px;
+        }
+
+        h2 {
+            @include font-size(36px);
+        }
+
+        p {
+            @include font-size(24px);
+        }
+
+        .button {
+            @include font-size(26px);
+        }
+    }
+
+    @media #{$mq-desktop-wide} {
+        .content {
+            width: $width-desktop;
+        }
+    }
+}
+
+html[dir="rtl"] #technologies > section {
+    @media #{$mq-tablet} {
+        h2 {
+            text-align: right;
+        }
+
+        p {
+            text-align: right;
+        }
+    }
+}
+
+#vr {
+    .content {
+        @include background-size(220px 200px);
+        background-image: url('/media/img/mozorg/technology/a-frame.svg');
+        background-position: top center;
+        background-repeat: no-repeat;
+        padding-top: 220px;
+    }
+
+    footer {
+        background: #00a199;
+        border-bottom: 2px solid #006c66;
+    }
+
+    @media #{$mq-phone-wide} {
+
+        .content {
+            @include background-size(260px 236px);
+            padding-top: 256px;
+        }
+    }
+
+    @media #{$mq-tablet} {
+
+        .content {
+            @include background-size(360px 327px);
+            background-position: top left;
+            padding-top: 0;
+        }
+    }
+
+    @media #{$mq-desktop} {
+
+        .content {
+            @include background-size(425px, 386px);
+        }
+    }
+}
+
+#gaming {
+    background-color: #d6f1f5;
+    padding-top: 236px;
+    z-index: 0;
+
+    .inner-container {
+        padding-top: 20px * 2;
+    }
+
+    footer {
+        background: #00b4cc;
+        border-bottom: 2px solid #008799;
+    }
+
+    &:after {
+        background-color: #a3f3fe;
+        background-image: url('/media/img/mozorg/technology/gaming-yeti.svg'),
+                          url('/media/img/mozorg/technology/gaming-background.svg');
+        background-position: bottom -60px right -30px, bottom right;
+        background-repeat: no-repeat, no-repeat;
+        -webkit-background-size: contain, cover;
+                background-size: contain, cover;
+        content: '';
+        height: 256px;
+        position: absolute;
+        right: 0;
+        top: 0;
+        width: 100%;
+        z-index: -1;
+    }
+
+    @media #{$mq-tablet} {
+        padding-top: 0;
+
+        &:after {
+            background-position: bottom -20px left 100px, bottom right;
+            height: 100%;
+            width: 50%;
+        }
+    }
+
+    @media #{$mq-desktop} {
+        &:after {
+            background-position: bottom -20px left 250px, bottom right;
+        }
+    }
+}
+
+#iot {
+    padding-top: 216px;
+    z-index: 0;
+
+    .inner-container {
+        padding-top: 20px * 2;
```

Comment:
Doh yeah, these are remnants of a mass find/replace for `@baseLine` when converting from Sandstone to Pebbles. Will update.